sample/app/build.gradle

```diff
@@ -66,7 +66,9 @@ staticAnalysis {
 
     ktlint {
         android.set(true)
-        reporters = [ReporterType.CHECKSTYLE]
```

Comment:
In order to do this, I had to put `ReporterType` in some other Gradle file. That was hacky. Glad that they fixed this. Can you search for `RegularFileProperty` in the sample folder and remove it? Thanks.